From 4de9a053530e460a01af2c562564e5e3c9da4871 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@nxp.com>
Date: Fri, 12 Jan 2018 10:56:11 -0600
Subject: [PATCH 3196/5242] MLK-17369: soc:imx8qm/qxp: Add controls for
 display controller resets

commit  9d58cdc0439f0213744d0ead2a5ee68dde17d46c from
https://source.codeaurora.org/external/imx/linux-imx.git

"
commit cfdb9821531da523fd1f01536eb67c8b8451477f
Author: Oliver Brown <oliver.brown@nxp.com>
Date:   Tue Jan 2 07:46:06 2018 -0600

    dc: Add controls for display controller resets.
"

Signed-off-by: Oliver Brown <oliver.brown@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/soc/imx8/sc/types.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/soc/imx8/sc/types.h b/include/soc/imx8/sc/types.h
index 8d8b046..4192809 100644
--- a/include/soc/imx8/sc/types.h
+++ b/include/soc/imx8/sc/types.h
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2016 Freescale Semiconductor, Inc.
- * Copyright 2017 NXP
+ * Copyright 2017-2018 NXP
  *
  * SPDX-License-Identifier:     GPL-2.0+
  */
-- 
1.7.9.5

